spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkReaderDeletes.java

```diff
@@ -130,16 +140,19 @@ protected Table createTable(String name, Schema schema, PartitionSpec spec) {
     TableOperations ops = ((BaseTable) table).operations();
     TableMetadata meta = ops.current();
     ops.commit(meta, meta.upgradeToFormatVersion(2));
-    if (vectorized) {
+    table.updateProperties().set(TableProperties.DEFAULT_FILE_FORMAT, format).commit();
+    if (format.equals("parquet") && vectorized) {
       table
           .updateProperties()
           .set(TableProperties.PARQUET_VECTORIZATION_ENABLED, "true")
           .set(
               TableProperties.PARQUET_BATCH_SIZE,
               "4") // split 7 records to two batches to cover more code paths
           .commit();
-    } else {
+    } else if (format.equals("parquet")) { // in this case, non-vectorized
       table.updateProperties().set(TableProperties.PARQUET_VECTORIZATION_ENABLED, "false").commit();
+    } else if (format.equals("orc")) { // we only have non-vectorized for orc in our parameters
+      table.updateProperties().set(TableProperties.ORC_VECTORIZATION_ENABLED, "false").commit();
```

Comment:
Minor suggestion: this will cover all cases, and no comment is needed.
```
    if (format.equals("parquet")) {
      table.updateProperties().set(TableProperties.PARQUET_VECTORIZATION_ENABLED, String.valueOf(vectorized)).commit();
      if (vectorized) {
        table.updateProperties().set(TableProperties.PARQUET_BATCH_SIZE, "4").commit();
      }
    } else if (format.equals("orc")) {
      table.updateProperties().set(TableProperties.ORC_VECTORIZATION_ENABLED, String.valueOf(vectorized)).commit();
      if (vectorized) {
        table.updateProperties().set(TableProperties.ORC_BATCH_SIZE, "4").commit();
      }
    }
```

Reply:
Thanks for the suggestion. I have adopted a variant of it.